### ISSUE TITLE
find maxlen then padding in all seq to prevent tf tracing which may become computationally expensive

### ISSUE THREAD

**Issue by laserene** [la-serene/English-German-Translator#4]

**Comment by laserene** [comment]
training dataset is modified as ((english_inputs, german_inputs), german_targets) to conform tf.keras.models.Model's fit() method.
